test_inti.py

```
@@ -0,0 +1,384 @@
+# test_integration.py (ADD NEW FILE)
+#!/usr/bin/env python3
```

Ruff (0.12.2)

Shebang is present but file is not executable

(EXE001)